test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommandTests/RestoreCommandTests.cs

```diff
@@ -454,6 +454,61 @@ public async Task RestoreCommand_WhenSwitchingBetweenLowercaseSettings_LockFileA
             }
         }
 
+        [Fact]
+        public async Task RestoreCommand_ProjectJsonProjectType_LogsNU1016ErrorAsync()
+        {
+            // Arrange
+            var project1Json = @"
+            {
+              ""version"": ""1.0.0"",
+              ""description"": """",
+              ""authors"": [ ""author"" ],
+              ""tags"": [ """" ],
+              ""projectUrl"": """",
+              ""licenseUrl"": """",
+              ""frameworks"": {
+                ""netstandard1.3"": {
+                    ""dependencies"": {
+                    }
+                }
+              }
+            }";
+
+            using var workingDir = TestDirectory.Create();
```

Comment:
nit: for future tests, use SimpleTestPathContext to get all these files configuration